WcaOnRails/app/controllers/admin/results_controller.rb

```diff
@@ -10,7 +10,7 @@ def posting_index
           render :posting_index
         end
         format.json do
-          @pending_competitions = Competition.pending_posting
+          @pending_competitions = Competition.pending_posting.sort_by { |comp| comp.results_submitted_at }
```

Comment:
I think we can let mysql do the sorting here and put `order(announced_at: :desc)` either here or right in the `pending_posting` scope.

Reply:
Thanks, done. (Hope you meant `.order(results_submitted_at: :asc)`)

Reply:
I did, thanks for fixing my mind :)